botocore/httpsession.py

```diff
@@ -45,6 +45,9 @@ def get_cert_path(verify):
     if verify is not True:
         return verify
 
+    cert_path = where()
```

Comment:
Probably makes sense to return 'cert_path' instead of calling where() twice.

Reply:
🤦🏻 forgot to push that, corrected. Thanks!